docs/content/development/extensions-core/lookups-cached-global.md

```diff
@@ -8,19 +8,25 @@ layout: doc_page
 Lookups are an <a href="../experimental.html">experimental</a> feature.
 </div>
 
-Make sure to [include](../../operations/including-extensions.html) `druid-namespace-lookup` as an extension.
+Make sure to [include](../../operations/including-extensions.html) `druid-lookups-cached-global` as an extension.
 
 ## Configuration
+<div class="note caution">
+Static configuration is no longer supported. Only cluster wide configuration is supported
```

Comment:
I dont think we need to do that as QTL as been labeled experimental and we clearly state we can and will change the API at any time.